content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -260,7 +260,51 @@ unifiedControlPlaneImage: ""
 ### Adding kube-proxy parameters {#kube-proxy}
 
 For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
+- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+
+### Using ipvs mode for kube-proxy
+
```

Comment:
please add link to this page:
https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md

and remove the rest of additions in this PR.

we need the two documents to compliment each other.
this introduces duplicate information as per the discussion here:
https://github.com/kubernetes/kubernetes/pull/68075

Reply:
ok Thanks